### DISCUSSION TITLE
Please help.  I've tried everything but still get "No module named 'pymatting_aot.aot'"

### DISCUSSION THREAD

**Discussion by bcizle** [danielgatis/rembg#101] [Q&A]
I'm on windows 10
I started on anaconda 3.8
Then a standalone Python 3.9
Got the right torch per the readme 
![image](https://user-images.githubusercontent.com/7064821/116657947-7d50fb80-a94c-11eb-944b-8155ed4b734e.png)
Installed c++ 142

All with the same result "No module named 'pymatting_aot.aot'"

Any help is appreciated.

**Comment by Sirofjelly** [comment]
have the same problem... were you able to fix it?

**Reply by bcizle** [reply]
Nope. :(

**Reply by hrishi2016** [reply]
did you guys solved it?

**Comment by ibowman** [comment]
Can you try running:

```
import sys
print(sys.executable) 
```

before the import to verify that you are running python from the correct environment?

**Reply by danielgatis** [reply]
https://github.com/danielgatis/rembg/issues/35

**Reply by ERMGLCAN** [reply]
tiene problemas para ejecutarlo en windows
